html/semantics/embedded-content/the-img-element/historical-progress-event.window.js

```diff
@@ -0,0 +1,16 @@
+async_test(t => {
+  const img = new Image();
+  t.add_cleanup(() => img.remove());
+  img.onloadstart = img.onprogress = img.onloadend = t.unreached_func("no progress events supported");
```

Comment:
```suggestion
  img.onloadstart = img.onprogress = img.onloadend = t.unreached_func("progress event fired");
```

The error message will be something like `assert_unreached: ${description} Reached unreachable code`, so I think that describing what happened leads to better messages.